src/core/virtual_sites/VirtualSitesRelative.hpp

```diff
@@ -20,38 +20,45 @@
 #ifndef VIRTUAL_SITES_VIRTUAL_SITES_RELATIVE_HPP
 #define VIRTUAL_SITES_VIRTUAL_SITES_RELATIVE_HPP
 
-#include "config.hpp" 
+#include "config.hpp"
 #ifdef VIRTUAL_SITES_RELATIVE
 
-#include "virtual_sites.hpp"
-#include "particle_data.hpp"
 #include "communication.hpp"
+#include "particle_data.hpp"
+#include "virtual_sites.hpp"
+
+/** @brief Virtual sites implementation for rigid bodies */
+class VirtualSitesRelative : public VirtualSites {
+public:
+  VirtualSitesRelative(){};
```

Comment:
Not needed.

Reply:
Even if it was necassary it should have been
```cpp
VirtualSitesRelative() = default;
```

Reply:
I guess this is a matter of style as the class has no user-defined special functions. If kept, the `= default` version makes the intent clearer.

Reply:
[C.80: Use `=default` if you have to be explicit about using the default semantics](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c80-use-default-if-you-have-to-be-explicit-about-using-the-default-semantics)

Reply:
i haven't touched that part besides formatting